### PR TITLE
fix(v1): update client library recommendations for v1, v3

### PR DESCRIPTION
**Pull request by jstirnaman**
- Updates recommended client libraries to use with v1
- Adds v3 compat information

### CODE REVIEW

**Comment by jstirnaman** [comment]
@sanderson Should we update this warning to say v3 or make it specific to OSS?

image

**Comment by sanderson** [comment]
I think we should probably make it OSS specific. I think if you update line 3 in `layouts/partials/article/stable-version.html` with the following, it should make it so it says "InfluxDB OSS v2".

```html
{{ $productName := cond (isset (index .Site.Data.products $product) "altname") (index .Site.Data.products $product).altname (index .Site.Data.products $product).name }}
```

**Comment by jstirnaman** [comment]
> I think we should probably make it OSS specific. I think if you update line 3 in `layouts/partials/article/stable-version.html` with the following, it should make it so it says "InfluxDB OSS v2".
> 
> ```
> {{ $productName := cond (isset (index .Site.Data.products $product) "altname") (index .Site.Data.products $product).altname (index .Site.Data.products $product).name }}
> ```

Thanks! I looked and was afraid I'd insert it somewhere else without knowing it.

**Comment by jstirnaman** [comment]
> > I think we should probably make it OSS specific. I think if you update line 3 in `layouts/partials/article/stable-version.html` with the following, it should make it so it says "InfluxDB OSS v2".
> > ```
> > {{ $productName := cond (isset (index .Site.Data.products $product) "altname") (index .Site.Data.products $product).altname (index .Site.Data.products $product).name }}
> > ```
> 
> Thanks! I looked and was afraid I'd insert it somewhere else without knowing it.

✅ Can you review this?

File: layouts/partials/article/stable-version.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {{ $productPathData := findRE "[^/]+.*?" .RelPermalink }}
 {{ $product := index $productPathData 0 }}
-{{ $productName := (index .Site.Data.products $product).name }}
+{{ $productName := cond (isset (index .Site.Data.products $product) "altname") (index .Site.Data.products $product).altname (index .Site.Data.products $product).name }}
```
Comment (sanderson) [inline layouts/partials/article/stable-version.html:3]: I didn't test this. Is it working?

**Review by sanderson — APPROVED** [review]
Looks good. I didn't test the template update. Just want to make sure it's working.

**Pull request merged**